README.md

```diff
@@ -57,6 +57,12 @@ To use simply include the Javascript bundle on your page. All elements with the
 
 `force-show`: Force showing the tooltip which can be used if you are trying to implement accessability concerns such as show with keyboard. (You must put tab-index="0" on the badge div to have it selectable by keyboard)
 
+`vertical-compact`: Shows the badge in vertical format without labels, only icons and numbers.
+
+`hide-retractions`: Force to hide retractions data.
+
+`hide-notices`: Force to hide notices data.
```

Comment:
I know its a little nit picky but I prefer the semantics of 
```
show-retractions
show-notices
```
that default to true... 
The reason is primarily around not wanting to send the message that we encouraging hiding these things heh.

I understand that makes things slightly more annoying in main.js but should be easy enough to change.